### ISSUE TITLE
How often to scan for docker images changes

### ISSUE THREAD

**Issue by rimusz** [rusenask/keel#25]
an option to set how often to scan docker repository for image changes

**Comment by rusenask** [comment]
on `polling` branch default value is `const KeelPollDefaultSchedule = "@every 1m` but it's configurable and uses https://github.com/rusenask/cron so you can define proper schedules like:
```
Entry                  | Description                                | Equivalent To
-----                  | -----------                                | -------------
@yearly (or @annually) | Run once a year, midnight, Jan. 1st        | 0 0 0 1 1 *
@monthly               | Run once a month, midnight, first of month | 0 0 0 1 * *
@weekly                | Run once a week, midnight on Sunday        | 0 0 0 * * 0
@daily (or @midnight)  | Run once a day, midnight                   | 0 0 0 * * *
@hourly                | Run once an hour, beginning of hour        | 0 0 * * * *
```

**Comment by rimusz** [comment]
can that be set via env var passed to container?

**Comment by rusenask** [comment]
currently (it's not merged yet) you need to set a label:
`keel.sh/pollSchedule=@every 10m`

would you want to override default to something else? definitely a reasonable feature then

**Comment by rimusz** [comment]
nice, so with the annotation I can set different time intervals for each deployment.?

**Comment by rusenask** [comment]
yeah, polling trigger sets up custom cron schedules (if specified) for all watched deployments, it might have been slightly overengineered though. Current limitation is however that if you have deployment A and deployment B that rely on the same image - they will share the same schedule (the last one that starts watching will update schedule).

**Comment by rusenask** [comment]
available in 0.3.0 release,
more info https://github.com/rusenask/keel#polling
apparently had to use annotations to be able to use more symbols `@every 10m`,
example available here https://github.com/rusenask/keel#polling

**Comment by rimusz** [comment]
we can close this one
